### PR TITLE
Add some methods to get servers and check connection

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: openmanage/pom.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
 		<jackson.version>2.10.0</jackson.version>
 		<maven-jar-plugin.version>3.1.1</maven-jar-plugin.version>
 		<jacoco.version>0.7.9</jacoco.version>
+		<apache.version>9.0.37</apache.version>
```
Comment (yixingjia) [inline openmanage/pom.xml:29]: double check.

Comment (Pengpengwanga) [reply openmanage/pom.xml]: removed

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java [hunk 2]
```diff
@@ -36,35 +48,61 @@ public OpenManageAPIClient(FacilitySoftwareConfig config) {
...
    }
 
-   public void setPassword(String password) {
-      this.password = password;
+   public void getToken() {
+      HttpEntity<Object> postEntity = new HttpEntity<Object>(this.authInfo, RestTemplateBuilder.getDefaultHeader());
+      ResponseEntity<Void> entity = this.restTemplate.exchange(getServiceEndPoint() + SessionUri,
+            HttpMethod.POST, postEntity , Void.class);
+      if(entity.getStatusCode().is2xxSuccessful() &&
+            !entity.getHeaders().get("x-auth-token").isEmpty()) {
```
Comment (yixingjia) [inline openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java:70]: define a constant variable.

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java [hunk 3]
```diff
@@ -36,35 +48,61 @@ public OpenManageAPIClient(FacilitySoftwareConfig config) {
...
+      if (entity.hasBody()) {
+         serversResult = entity.getBody();
+      }
+      return serversResult;
+   }
 
-   public boolean checkConnection() {
-      //TODO
-      return true;
+   public DevicesResult<Chassis> getChassis(int skip, int limit) {
```
Comment (yixingjia) [inline openmanage/src/main/java/com/vmware/flowgate/openmanage/client/OpenManageAPIClient.java:94]: check whether can use generic type.

**Review by yixingjia — APPROVED** [review]

**Pull request merged**